app/views/layouts/metadata_presenter/application.html.erb

```diff
@@ -38,8 +38,7 @@
       </main>
     </div>
     <%= render template: 'metadata_presenter/footer/footer' %>
-
-    <%= javascript_pack_tag 'application' %>
+    <%= javascript_pack_tag 'runner_application' %>
```

Comment:
the preview mechanism in the editor offloads the layout responsibility to the presenter. would this be ok in those instances?

Reply:
Yes. It's a little disappointing I cannot easily show the partner action in this PR but there is are additional changes (see other PRs in waiting) to both Editor and Runner that make sure the requested file is in place. It's not the nicest solution but it will prevent the JS error we get in Preview that blocks any valid JS from running after this point.